api_demo/ts_field.py

```diff
@@ -0,0 +1,3 @@
+from datetime import datetime
+
+print(datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
```

Comment:
Why do we need this?

Reply:
I use this file when I need to test some functions in python, when I don't remember, how does it work. I can remove this file and test functions in another place